sale_quotation_capable_vendor_rfq/models/sale_order.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    purchase_order_ids = fields.One2many(
```

Comment:
We are actually creating RFQs not purchase orders. Also here we can run into the same issue when some other module will reimplement the same fields. Because its name is to generic.
So `rfq_ids` would be better.